docs/cloudfoundry/cf-per-route-virtuals.rst

```diff
@@ -55,6 +55,12 @@ You can use as many Service Plans as you need to define the BIG-IP services your
 
    \
 
+   .. warning::
+
+      Make sure that you follow the proper Classless Inter-Domain Routing format (`CIDR format`_) when allocating IPs or networks for the :code:`tier2_ip_range` parameter otherwise errors will occur when configuring objects on the BIG-IP device. See `Virtual Server parameters`_ under the Source Address table entry for more information on this parameter. A good rule of thumb is making sure that your network prefixes fall onto CIDR block boundaries like 172.0.0.0/24 for IPv4.
```

Comment:
```
It's important to use the proper `CIDR format`_ when allocating IP addresses or networks for the :code:`tier2_ip_range` config parameter (for example, :code:`172.0.0.0/24` for IPv4). Failure to use the proper format may result in errors when the |cfctlr| configures BIG-IP objects.

See `Overview of the Standard Virtual Server`_ on AskF5 for more information. 
```

Reply:
Thanks Jodie I will make this change.

Reply:
We should keep the language referring to boundaries; 10.105.175.245/30 is properly formatted, but this will fail on the BIG-IP as it's not referencing an address at the boundary - the valid input would be 10.105.175.244/30. The support link does mention it should be a network but I think that's easily missed.